File: testdata/min_max_sizes/expected.dart
Widget build(BuildContext context) {
  return   Container(
    width: double.infinity,
    padding: EdgeInsets.all(12.0),
    margin: EdgeInsets.all(0.0),
    child:     Row(
      mainAxisAlignment: MainAxisAlignment.start,
      crossAxisAlignment: CrossAxisAlignment.start,
      children: [
        Expanded(
          flex: 1,
          child:           Container(
            height: 80.0,
            padding: EdgeInsets.all(8.0),
            margin: EdgeInsets.all(0.0),
            constraints: BoxConstraints(
              maxWidth: 100.0,
            ),
            color: Color.fromRGBO(251, 180, 174, 1.0),
            alignment: Alignment.center,
            child: Text('capped',
              style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
            ),
          )
        ),
        Expanded(
          flex: 1,
          child:           Container(
            height: 80.0,
            padding: EdgeInsets.all(8.0),
            margin: EdgeInsets.all(0.0),
            color: Color.fromRGBO(179, 205, 227, 1.0),
            alignment: Alignment.center,
            child: Text('free',
              style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
            ),
          )
        ),
        Expanded(
          flex: 1,
          child:           Container(
            height: 80.0,
            padding: EdgeInsets.all(8.0),
            margin: EdgeInsets.all(0.0),
            constraints: BoxConstraints(
              minWidth: 200.0,
            ),
            color: Color.fromRGBO(204, 235, 197, 1.0),
            alignment: Alignment.center,
            child: Text('wide',
              style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
            ),
          )
        ),
      ],
    )
  )
;
}

File: testdata/min_max_sizes/expected.swift
struct ContentView: View {
    public var body: some View {
        HStack(alignment: .top, spacing: 8.0) {
            Text("capped")
                .font(.system(size: 26))
                .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                .frame(width: nil, height: 80.0)
                .frame(minWidth: nil, maxWidth: 100.0, minHeight: nil, maxHeight: nil)
                .padding(8.0)
                .background(Color(red: 0.98, green: 0.71, blue: 0.68))
            Text("free")
                .font(.system(size: 26))
                .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                .frame(width: nil, height: 80.0)
                .frame(maxWidth: .infinity)
                .padding(8.0)
                .background(Color(red: 0.70, green: 0.80, blue: 0.89))
            Text("wide")
                .font(.system(size: 26))
                .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                .frame(width: nil, height: 80.0)
                .frame(minWidth: 200.0, maxWidth: nil, minHeight: nil, maxHeight: nil)
                .frame(maxWidth: .infinity)
                .padding(8.0)
                .background(Color(red: 0.80, green: 0.92, blue: 0.77))
        }
        .frame(minWidth: nil, maxWidth: .infinity, minHeight: nil, maxHeight: nil, alignment: .topLeading)
        .padding(12.0)
        .background(Color(red: 0.11, green: 0.11, blue: 0.17))
    }
}
